runtime/src/main/java/org/corfudb/runtime/collections/StreamingMetrics.java

```diff
@@ -0,0 +1,34 @@
+package org.corfudb.runtime.collections;
+
+import com.codahale.metrics.MetricRegistry;
+import com.google.gson.JsonObject;
+
+/**
+ * List of metrics captured for a stream listener.
+ * <p>
+ * Created by WenbinZhu on 11/23/20.
+ */
+public class StreamingMetrics {
+
+    private final String listenerId;
+
+    private final TimingHistogram deliveryDuration;
```

Comment:
Is this class only required for logging purposes? If the these stats can't be accessed externally and only used for logging, then I would suggest converting it to a proper Metric so that we can collect it and log it as within our metrics performance library. 

This metric can be kept registered and kept in the `StreamSubscription` instance.

Reply:
Also, note that `TimingHistogram` uses a EDR histogram (histogram) underneath which is not really accurate when reporting these stats in a short interval.

Reply:
Done.